src/d3d9/d3d9_common_texture.h

```diff
@@ -366,6 +366,42 @@ namespace dxvk {
     void PreLoadAll();
     void PreLoadSubresource(UINT Subresource);
 
+    void AddUpdateDirtyBox(CONST D3DBOX* pDirtyBox, uint32_t layer) {
+      if (pDirtyBox) {
+        D3DBOX box = *pDirtyBox;
+        if (box.Right <= box.Left
+          || box.Bottom <= box.Top
+          || box.Back <= box.Front)
+          return;
+
+        D3DBOX& updateBox = m_updateDirtyBoxes[layer];
+        if (updateBox.Left == 0 && updateBox.Right == 0
```

Comment:
This degenerate test could be simpler.

We could just do Left == Right probably

Reply:
This is not a degenerate test. It tests whether to use the entire box passed in.